airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This module contains Google Dataprep hook.
+"""
+import os
+
+import requests
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+TOKEN = os.environ["DATAPREP_TOKEN"]
+URL = "https://api.clouddataprep.com/v4/jobGroups/"
+
+
+class GoogleDataprepHook:
+    """
+    Hook for connection with Dataprep API.
+
+    """
+
+    def __init__(self) -> None:
```

Comment:
```suggestion
    def __init__(self, dataprep_conn_id: str = "dataprep_conn_id") -> None:
```

Reply:
I am not using this var right know because there is no connection with dataprep like with GCP for example. I next PR I wanted to make permanent connection with Dataprep. Wdyt?

Reply:
When introducing new operators/hooks to Airflow we should thrive to provide code that meets community standards. This includes providing authentication methods with which Airflow users are familiar. And that's why I think we should do this in single PR. Otherwise, users will be confused.

Reply:
Adding new connection is simple as adding two lines in `airflow/models/connection.py`